code/modules/client/client procs.dm

```diff
@@ -362,14 +362,17 @@
 
 /client/proc/log_client_to_db()
 
-	if( IsGuestKey(src.key) )
+	if( IsGuestKey(key) )
 		return
 
+	// Isn't the ckey sanitized by default?
+	var/sql_ckey = sanitizeSQL(ckey)
```

Comment:
The ckey is just the client's key but all in lowercase, with everything restricted to alphanum and dashes+underscores